src/main/app/case/to-api-format.test.ts

```diff
@@ -6,7 +6,7 @@ describe('to-api-format', () => {
   const results: Partial<Case> = {
     gender: Gender.MALE,
     sameSex: Checkbox.Checked,
-    relationshipDate: { year: '1900', month: '1', day: '4' },
+    relationshipDate: { year: '1900', month: '01', day: '04' },
```

Comment:
does this matter? the example of a date shows single digits.

Reply:
To be consistent with what's stored in the API and for validating the current session against the API response, I've changed the `session.userCase` to match what should be stored in the API.

Reply:
I think it was this way because the frontend provides it as 1900, 1, 4 which the API doesn't support so we wanted to make sure padding was applied.

Reply:
👍 I've refactored this so that the `fromApiFormat` will drop any padded zeros in 559b32b.